NDTensors/src/lib/SetParameters/src/specify_parameters.jl

```diff
@@ -28,12 +28,10 @@ function specify_parameters(type::Type, parameter...)
   return set_parameters(specify_parameters, type, Position(1), parameter...)
 end
 
-function specify_parameters(type::Type, parameters::Tuple)
-  new_type = type
-  for i in 1:(length(parameters))
-    new_type = specify_parameters(new_type, Position(i), parameters[i])
-  end
-  return new_type
+function specify_parameters(
+  type::Type, parameters::Tuple, starting_position::Position=Position(1)
+)
+  return set_parameters(specify_parameters, type, starting_position, parameters)
```

Comment:
I'm confused, isn't this the same functionality as the one defined on line 20 of this file?

Reply:
Okay sorry let me look at this file again.

Reply:
So it looks like this function is not necessary. I think I added it for the `Dense` `set_parameter` functions and since we have changed their definitions the function is not used/necessary. I have removed it from the PR